packages/web3-react/src/components/Pages/Profile/index.tsx

```diff
@@ -18,23 +18,18 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 import React, { useEffect, useState } from 'react'
 import { useConnectSettingContext } from '../../../contexts/alephiumConnect'
 
-import { PageContent, ModalBody, ModalContent, ModalH1 } from '../../Common/Modal/styles'
+import { PageContent, ModalContent, ModalH1 } from '../../Common/Modal/styles'
 import Button from '../../Common/Button'
 
 import { DisconnectIcon } from '../../../assets/icons'
 import CopyToClipboard from '../../Common/CopyToClipboard'
 import { useAccount } from '../../../hooks/useAccount'
 import { truncatedAddress } from '../../../utils'
-import { useBalance } from '../../../hooks/useBalance'
-import { AnimatePresence } from 'framer-motion'
-import { Balance, BalanceContainer, LoadingBalance } from './styles'
-import { prettifyAttoAlphAmount } from '@alephium/web3'
 import { useConnect } from '../../../hooks/useConnect'
 
 const Profile: React.FC<{ closeModal?: () => void }> = ({ closeModal }) => {
   const context = useConnectSettingContext()
   const account = useAccount()
-  const { balance } = useBalance()
   const { disconnect } = useConnect({
```

Comment:
We have to remove `useBalance` from here, because `Profile` is not within the `AlephiumBalanceProvider` component tree.

Reply:
We should add `AlephiumBalanceProvider` into `AlephiumWalletProvider`. Even if the app does not consume the provider, the overhead is insignificant.

Reply:
But I agree that we should remove `useBalance` from `Profile`. Showing just ALPH is not that necessary